lmdeploy/turbomind/turbomind.py

```diff
@@ -765,13 +782,15 @@ async def async_stream_infer(self,
                     outputs = (status, output[:-1].tolist(), len_)
                 else:
                     outputs = (status, output.tolist(), len_)
+            if outputs[-1] < prev_len and not finish:
```

Comment:
When will `outputs[-1] < prev_len`?

Reply:
If the consumer is accidentally faster than the producer. Then continuous two elements will be fetched one by one. The previous fetched one could be longer than the latter one.